### DISCUSSION TITLE
8/3 <초조한 전망대> 네번째 모임 후기: AI 뉴스 및 모임 방향성 논의

### DISCUSSION THREAD

**Discussion by statsops** [Pseudo-Lab/Anxious-Lookout#20] [General]
# <초조한 전망대> 네번째 모임 후기 : AI 뉴스 소개와 앞으로의 방향성 논의

* 분위기: 조금은 조용하고 진중하다.
* 참석 멤버: 정한결, 이홍규, 이유신, 강호현, 함경표, 정재희, 박주영, 강동원 (8명)
* 시간 : 8월 4일 5시 ~ 6시 40분
* 장소 : 디스코드 ROOM-YL

<img width="500" alt="사진" src="https://github.com/user-attachments/assets/507ceaf0-be2c-48f3-840a-30f0dd314a62">

지난주 지식교류회를 마치고, 다시 처음의 마음으로 돌아간 모임입니다. 총 두 개의 세션으로 나누어 진행했습니다. 

1부는 기존에 진행했던 AI 뉴스 기사를 가져와서 발표하고 논의하였습니다. 2부는 <초조한 전망대>의 운영 방식에 대해 논의해보고 방향을 설정해보는 시간을 가졌습니다.

## 1부: AI 뉴스 기사 논의

각자의 AI 뉴스 기사를 가져와서 함께 논의하는 시간을 가졌습니다.

### 1. 한결님: maestro 이야기

https://levelup.gitconnected.com/deepseek-coder-llama-3-how-i-build-application-with-one-prompt-05a77b02cace (유료 글입니다. 전부 보려면 돈을 내야합니다.)
Maestro라는 Python기반 프레임워크를 가지고, snake game을 만드는 과정이 있습니다. 개발에 AI를 접목하고, 쉽게 게임을 만들어볼 수 있는 과정이 있었네요.

![image](https://github.com/user-attachments/assets/3e942082-785e-400a-86c7-ffc3781ee814)

### 2. 유신님: Sun Jimung 교수님의 의료 AI 강의 후기
(https://scholar.google.com/citations?user=9jmmp5sAAAAJ&hl=en&oi=ao)
홍콩 교수님, Sun님의 강의를 듣고 와서 그 분의 논문 목록을 보여주셨습니다.
의료 AI에서 유명한 Sun님의 논문 목록을 보았습니다. 의학 논문의 높은 인용수에 한번 놀라고, 들은 강의에서 프로토콜을 원하는 형태로 구성하고 생성하는 방법에 대해 알게 되었다고 합니다. 프로토콜은, 환자의 대응 방식 등 procedure를 의미한다고 합니다. 그게 생성형으로 가능하다면 점차 병원 내 모습에도 영향을 줄 수 있겠네요.

### 3. 경표님:  1년 동안 LLM과 함께 구축하며 배운 점 REVIEW
https://news.hada.io/topic?id=15268
1년간 LLM으로 구축해본 분의 회고를 보면서, 공감할 부분을 공감해주고 본인의 경험을 곁들여 리뷰해주셨습니다. 아주 꼼꼼하게 리뷰를 해주셨네요. 많은 인사이트를 얻어갈 수 있었습니다. 꼭 한번 전체적으로 정독해보기를 추천합니다.

얀님의 원본 블로그: https://eugeneyan.com

### 4. 재희님: AI시대에서 인간의 역할 논의
재희님은 지난번 논의한 내용을 다시 가져와서, AI의 발전으로 인해 사람의 일자리를 모두 대체하지 않을까?에 대한 주제로 가벼운 논의를 진행했습니다.

### 5. 호현님: 재미있는 AI 로봇 뉴스
짧은 AI 기반 로봇의 영상을 보면서, 사람이 되고 싶은 AI?라는 제목으로 가볍게 의견을 냈습니다.

[![YouTube Video](https://img.youtube.com/vi/r6EttBnT1bI/0.jpg)](https://www.youtube.com/watch?v=r6EttBnT1bI)

## 2부: <초조한 전망대> 운영 논의
2부는 지난주 지식교류회 이후 나온 다양한 초조한 전망대의 앞으로 운영 방식에 대해 논의하는 시간을 가졌습니다. 한결님이 잘 정리해주신 내용을 바탕으로, 다른 분들의 피드백을 모아 정리해보고 결정하고자 합니다.

>  초조한 전망대 - 현재 문제 인식! 
> 
> 1. 온보딩 프로세스 누수- 들어오신 분들이 모임에 나오기까지, 그리고 나오고 활발하게 활동하는 과정에까지 이탈이 있습니다. (백프로 참여하게 만들 수는 물론 없지만) 설명이 불충분한 부분 등이 있을 겁니다. 참여자 입장을 물어보고, 프로세스 중 개선할 수 있는 부분을 찾아보겠습니다. 
> 2. 더 조직적 역할 정의 - 많은 분들이 이번 주 기사 올리시는걸 잊으신듯! 각자 뭘 해야 하는지 아직 습관이 안 된 상황으로 보입니다. 어떤 역할을 해 주어야 하는지 명확하게 정의해야 하겠습니다. 제가 리마인드 하겠습니다. 
> 3. 우리는 뭐 한다를 정립할 필요 -  초조한 전망대는 3개의 목표(정보 공유, 온보딩, 네트워킹)가 있습니다. 다 챙기려다 보니 좀 혼란스러워요. 그래서 정리를 위해 어제 대화를 나눴고, 그 내용을 아래처럼 정리해봤어요. 앞으로도 대화하며 방향을 변경하며 발전시킬 겁니다. 
>     1. “정보 공유”: 서로 유익한 기사를 금요일까지 모아오고, 토요일날 온라인에서 1분 요점 요약 & 토론합니다. 
>     2. 온라인 모임 (”네트워킹 + 온보딩”): 오래 계셨던 분들이 새로운 멤버들에게 붙어서 얘기해줍니다. (저의 개인적 목표는 모든 참여자가 기존 스터디 연결 or 자기만의 모임이나 플젝을 시작하게 해 주는 겁니다. ) 
>     3. 오프라인 (”네트워킹”): 직무별로 섞여서 노는 시간, 좀 더 친해지는 시간이 되어야겠습니다. 또 우리 하는거 보여주는 시간이기도 하고요.  
>     4. 결과물 만들기    
> 4. 비개발자면서 AI에 관심 있는 분들 챙기기 - API. 창구 역할도 놓치고 싶지는 않네요. 그 분들에게 닿는 것도 생각해보고 싶어요. 예를 들면 프롬프팅, 툴링 리뷰 등은 공통의 관심사가 될 수 있을 것 같아요. 월 1회 이런 세션을 가지는 것도 좋겠구요. 과하면 폐기해야겠지만요. 
> 
> 참여자별 현재 할일  
>     - 기본적으로 우리 맡은 일이 뭔지는 이 문서에서 정의합니다 (https://github.com/Pseudo-Lab/impatient-observatory/discussions/17)
>     - 반복성 todo: 
>         1. 금요일 자정까지 소개하고픈 글 올리기 
>         2. 토요일 5시에 디스코드에서 모이기
>         3. 운영진 업무 (위 문서 참조) 
>     - 일회성 todo: 
>         1. 소개글 - 다시 쓰고, 깃허브 외부에 배포 
>         2. where to follow - 업데이트, 분류해야겠음. 새로운 분들이 와서 쉽게 뒤져보는게 목표. 
>         3. onboarding missions - 애매하다 깃헙 부분 빼고 다시 써보자
>         4. 기획 - 다음 오프라인 모임 및 비개발자 분들과 어울릴 방안
> - 다음 모임때 도와주실 분들을 받아서 위의 일들을 좀 나누겠습니다!

**Comment by HangryDev** [comment]
호현님 깔끔하게 정리해주셔서 감사합니다! 👍 저 뉴스 마지막에 더 쭉 봤어야 하는데 ㅎㅎ 얼렁뚱땅 넘어간것 같아서 아쉬웠어요 끝나고
